dapr-spring-core/src/main/java/io/diagrid/spring/core/keyvalue/DaprKeyValueAdapter.java

```diff
@@ -122,13 +124,18 @@ public CloseableIterator<Entry<Object, Object>> entries(String keyspace) {
 
     @Override
     public void deleteAllOf(String keyspace) {
-        // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'deleteAllOf'");
+        var keyspaceFilter = getKeyspaceFilter(keyspace);
+        var sql = String.format(DELETE_BY_KEYSPACE_SQL_PATTERN, keyspaceFilter);
+        var meta = Map.of("sql", sql);
+
+        daprClient.invokeBinding(stateStoreBinding, "exec", null, meta).block();
     }
 
     @Override
     public void clear() {
+        var meta = Map.of("sql", DELETE_ALL_SQL);
```

Comment:
Is this necessary? Does this needs to clean up all the keys?

Reply:
If you think it is too dangerous we can leave it as `Unsupported....`. I took inspiration from `MapKeyValueAdapter`

Reply:
Yeah I wouldn't follow the in memory implementation but the Redis one..